docs/design-system.md

```diff
@@ -0,0 +1,73 @@
+
+## Theme
+
+With this update we get a full new color palette and theme. The structure is changing a little bit to try and keep things more organized. You can dig into the app [assets/src/design-system/theme](https://github.com/google/web-stories-wp/tree/main/assets/src/design-system) to see organization. Here’s brief overview of the organization to contextualize what you’re looking at.
+
+1. Colors, Typography, Borders, Breakpoints are in separate files and all added to the index’s `theme` that can be ingested at the root of each app. This file separation makes it easier to find theme values for developers who prefer to look in source code for the value they need.
+2. The new theme provides a dark and a light variant. The dark is the default. All colors are available regardless of variant, but the `theme.colors.bg` and `theme.colors.fg` are what you should gravitate towards using because the light and dark theme colors mirror each other. This will make dark and light mode functionality easier to handle down the road.
+3. Subdirectory called `helpers` - these are styles meant to be quick references for one off components in the editor and dashboard that aren’t in the design system.They’re also used in the design system for continuity.
+4. Subdirectory called `global` - these are exported as `ThemeGlobals` and should be added to app root code to handle global styles for both the dashboard and editor. Currently only the focus override exists here.
+5. Subdirectory called `constants` - these are constants that are used in both the dashboard and the editor (or needed also in the design system) so that we can have a single source of truth. You can import them via `THEME _CONSTANTS`. These include things like static widths of wordPress menus that all projects reference.
+
+It’s important to note that this is a living resource - I fully anticipate that we need to add to the theme as we begin to use it more robustly. The design system is being created iteratively as we go through implementing new designs. If there’s a new need for a helper, add it! If there’s a color lacking, add it in following the existing patterns. My goal here was to create a base that we can easily keep updating and avoid needless duplication.
```

Comment:
```suggestion
It’s important to note that this is a living resource - we fully anticipate the need to add to the theme as we begin to use it more robustly. The design system is being created iteratively as we go through implementing new designs. If there’s a new need for a helper, add it! If there’s a color lacking, add it in following the existing patterns. Our goal here was to create a base that we could easily keep updated to avoid needless duplication.
```